api/datastore/internal/datastoreutil/metrics.go

```diff
@@ -83,6 +84,18 @@ func (m *metricds) InsertCall(ctx context.Context, call *models.Call) error {
 	return m.ds.InsertCall(ctx, call)
 }
 
+func (m *metricds) UpdateCallStatus(ctx context.Context, appName, callID, status string, completedAt strfmt.DateTime) error {
+	span, ctx := opentracing.StartSpanFromContext(ctx, "ds_update_call_status")
+	defer span.Finish()
+	return m.ds.UpdateCallStatus(ctx, appName, callID, status, completedAt)
+}
+
+func (m *metricds) UpdateCallMetrics(ctx context.Context, appName, callID string, CPUUsage, MemoryUsage uint64) error {
+	span, ctx := opentracing.StartSpanFromContext(ctx, "ds_update_call_metrics")
+	defer span.Finish()
+	return m.ds.UpdateCallMetrics(ctx, appName, callID, CPUUsage, MemoryUsage)
```

Comment:
can we just have a `map[string]uint64` to store? this api will require frequent breaking for adding any stats.